src/NuGet.Core/NuGet.Packaging/Licenses/NuGetLicenseData.cs

```diff
@@ -9,11 +9,7 @@ namespace NuGet.Packaging.Licenses
     // Auto-Generated by the GenerateLicenseList tool. DO NOT EDIT this manually. Use the update script at $repositoryRoot/scripts/utils/UpdateNuGetLicenseSPDXList.ps1
     public static class NuGetLicenseData
     {
-        // The data source for this version was corrupted, so this file had to be manually fixed:
-        // Refer to https://github.com/spdx/LicenseListPublisher/issues/102. 
-        // When this files gets regenerated, this explanation will be removed. 
-        // This is to be used as a marker to confirm that the linked issue with the data source has been fixed.
-        public static string LicenseListVersion { get; } = "da7ecca";
+        public static string LicenseListVersion { get; } = "fafa781";
```

Comment:
Where is this number coming from? Just curious. Is there some db to lookup or tool generate it?

Reply:
If you look at the top, line 9 it refers to the script that generates it. 

https://github.com/NuGet/NuGet.Client/pull/4308/files#diff-d75dd4353277fb020fbb7a27b5c83af0e5b81dc2b60f098492cf092137484270R9


https://github.com/NuGet/NuGet.Client/blob/dev/scripts/utils/UpdateNuGetLicenseSPDXList.ps1 runs https://github.com/NuGet/NuGet.Client/blob/dev/test/TestExtensions/GenerateLicenseList/Program.cs

Link here: https://github.com/NuGet/NuGet.Client/blob/dev/scripts/utils/UpdateNuGetLicenseSPDXList.ps1#L35